Corrections to C3

kwlodarczyk opened the pull request:
Below are the corections of issues found while updating schemas for CTT:
- Changed the baseline interface of 4 new AM resources to be the schema for the specific resource, not the base collection schema, which does not have "rts" and "rts-m" properties
- In schemas for the AM resources changed "allOf" to "properties"
- In schemas for the AM resources replaced the reference to the non-existent oic.collection with references to oic.collection.properties and oic.collection.links.arrayoflinks
- Corrected an invalid reference in oic.r.body.location.temperature.json
- Corrected inconsistency property name in oic.r.bloodpressure  ("units" is schema, "unit" in RAML)

hongkicha commented:
@kwlodarczyk 
I agree with the changes except for "units" property changes.
Sorry for the inconsistency, but they should've been "units" not "unit". 
DMWG and HCP agreed to define all units-related properties as "units".
Could you make them consistent as "units"?

kwlodarczyk commented:
@hongkicha
I changed the following as you suggested:
- Reverted the change "units" => "unit" in oic.r.bloodpressure.json
- Changed "unit" to "units" in bloodPressure.raml

I've noticed that "unit" is still used in both schema an d RAML for oic.r.glucose.medication. Shouldn't it be also changed?

hongkicha commented:
Yes, it should've changed. Could you please make those changes, please?
Thank you in advance.

hongkicha commented:
@kwlodarczyk 
I heard from Mark that changes of data models are uni-directional, meaning only oneIoTa -> GitHub is valid.
GitHub -> oneIoTa doesn't work, so I will reflect all changes you made directly to oneIoTa.
Thank you for your comments.

kwlodarczyk commented:
@hongkicha
Ok. I've made the change just for reference.
